src/LeanCode.ContractsGenerator/Program.cs

```diff
@@ -11,14 +11,18 @@ public interface IOptions
 {
     public const string StdoutMarker = "-";
 
-    [Option('o', "output", Required = true, MetaValue = "FILE", HelpText = "The output file path. Use `-` to write the resulting Protobuf file to `stdout`.")]
+    [Option('o', "output", MetaValue = "FILE", HelpText = "The output file path. Use `-` to write the resulting Protobuf file to `stdout`.", Group = "output")]
     public string OutputFile { get; set; }
+
+    [Option("check-only", HelpText = "Check the contracts only. Do not generate output.", Group = "output")]
+    public bool CheckOnly { get; set; }
```

Comment:
Why `check-only` instead of `dry-run`? I feel like `dry-run` is a much more popular name for this functionality.

Reply:
I started with dry run but it occurred to me that that the intention here is not to dry run generation (there is nothing dry in not generating the output, compared to e.g. not applying changes), but to run the checks - that's why I called it `check only`.

Reply:
That's indeed true, I agree with the decision then